pages/linux/pveversion.md

```diff
@@ -9,4 +9,4 @@
 
 - Print Proxmox subpackage versions:
 
-`pveversion {{-v|--version}}`
+`pveversion {{-v|--verbose}}`
```

Comment:
According to <https://pve.proxmox.com/wiki/Command_Line_Tools> only `-v` is supported and not `--version` or `--verbose`?

Reply:
See this comment with the bottom image https://github.com/tldr-pages/tldr/pull/13975#issuecomment-2394165922

I'm unable to find the manpage anywhere on the internet. I spun up a proxmox virtual machine because it's the only place I can view the man page and test the command.